packages/notifications/src/styled/global-alert/StyledGlobalAlert.ts

```diff
@@ -7,72 +7,65 @@
   let backgroundColor;
   let foregroundColor;
   let anchorHoverColor;
   let anchorActiveColor;
-  let focusColor;
-
-  switch (props.alertType) {
-    case 'success':
-      borderColor = getColorV8('successHue', 700, props.theme);
-      backgroundColor = getColorV8('successHue', 600, props.theme);
-      foregroundColor = getColorV8('successHue', 100, props.theme);
-      anchorHoverColor = props.theme.palette.white;
-      anchorActiveColor = props.theme.palette.white;
-      focusColor = 'successHue';
-      break;
-
-    case 'error':
-      borderColor = getColorV8('dangerHue', 700, props.theme);
-      backgroundColor = getColorV8('dangerHue', 600, props.theme);
-      foregroundColor = getColorV8('dangerHue', 100, props.theme);
-      anchorHoverColor = props.theme.palette.white;
-      anchorActiveColor = props.theme.palette.white;
-      focusColor = 'dangerHue';
-      break;
-
-    case 'warning':
-      borderColor = getColorV8('warningHue', 400, props.theme);
-      backgroundColor = getColorV8('warningHue', 300, props.theme);
-      foregroundColor = getColorV8('warningHue', 800, props.theme);
-      anchorHoverColor = getColorV8('warningHue', 900, props.theme);
-      anchorActiveColor = getColorV8('warningHue', 1000, props.theme);
-      focusColor = 'warningHue';
-      break;
-
-    case 'info':
-      borderColor = getColorV8('primaryHue', 300, props.theme);
-      backgroundColor = getColorV8('primaryHue', 200, props.theme);
-      foregroundColor = getColorV8('primaryHue', 700, props.theme);
-      anchorHoverColor = getColorV8('primaryHue', 800, props.theme);
-      anchorActiveColor = getColorV8('primaryHue', 900, props.theme);
-      focusColor = 'primaryHue';
-      break;
+  let focusVariable;
+
+  if (['error', 'success'].includes($alertType)) {
+    const borderVariable =
+      $alertType === 'success' ? 'border.successEmphasis' : 'border.dangerEmphasis';
+    const backgroundVariable =
+      $alertType === 'success' ? 'background.successEmphasis' : 'background.dangerEmphasis';
+    const foregroundVariable =
+      $alertType === 'success' ? 'foreground.success' : 'foreground.danger';
+
+    borderColor = getColor({ variable: borderVariable, light: { offset: 100 }, theme });
+    backgroundColor = getColor({ variable: backgroundVariable, theme });
+    foregroundColor = getColor({ variable: foregroundVariable, light: { offset: -600 }, theme });
+    focusVariable =
+      $alertType === 'success' ? 'foreground.successEmphasis' : 'foreground.dangerEmphasis';
+    anchorHoverColor = theme.palette.white;
+    anchorActiveColor = theme.palette.white;
+  } else {
+    const borderVariable =
+      $alertType === 'warning' ? 'border.warningEmphasis' : 'border.primaryEmphasis';
+    const backgroundVariable =
+      $alertType === 'warning' ? 'background.warningEmphasis' : 'background.primaryEmphasis';
+    const foregroundVariable =
+      $alertType === 'warning' ? 'foreground.warning' : 'foreground.primary';
```

Comment:
Are these ternaries buying us much? Feels like the previous `switch` was more readable.

Reply:
Reverted. :)